api/redisfailover/v1alpha2/types.go

```diff
@@ -39,12 +39,14 @@ type RedisSettings struct {
 	Resources RedisFailoverResources `json:"resources,omitempty"`
 	Exporter  bool                   `json:"exporter,omitempty"`
 	Version   string                 `json:"version,omitempty"`
+	ConfigMap string                 `json:"configMap"`
 }
 
 // SentinelSettings defines the specification of the sentinel cluster
 type SentinelSettings struct {
 	Replicas  int32                  `json:"replicas,omitempty"`
 	Resources RedisFailoverResources `json:"resources,omitempty"`
+	ConfigMap string                 `json:"configMap"`
```

Comment:
This should be optional (adding the `omitempty`), so we allow using the default configuration.